zjit/src/options.rs

```diff
@@ -67,6 +76,8 @@ pub const ZJIT_OPTIONS: &'static [(&str, &str)] = &[
     ("--zjit-num-profiles=num",   "Number of profiled calls before JIT (default: 1, max: 255)."),
     ("--zjit-stats",              "Enable collecting ZJIT statistics."),
     ("--zjit-perf",               "Dump ISEQ symbols into /tmp/perf-{}.map for Linux perf."),
+    ("--zjit-log-compiled-iseqs=path", "Log compiled ISEQs to the specified file. The file will
+                                        be truncated on each run."),
```

Comment:
If you try `./miniruby --help`, this renders into:

```
  --zjit-log-compiled-iseqs=path
                  Log compiled ISEQs to the specified file. The file will
                                                          be truncated on each run.
```

which seems weird.

Because `--zjit-log-compiled-iseqs=path` is already long by itself, the help automatically moves the line to the next line, so let's start the description in the source code in the next line as well.

And it's still too long by the way, so you need to make it as short as (Note that the position of the `80-char limit` comment was actually wrong. I'll fix it separately):

```suggestion
    ("--zjit-log-compiled-iseqs=path",
                     "Log compiled ISEQs to the file. The file will be truncated."),
```